README.md

```diff
@@ -21,39 +29,43 @@ Additional description about the project and its features.
 
```

Comment:
This live demo link is broken. Kindly fix this or remove it. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.